### PR TITLE
improve phpdoc of cache classes

### PR DESCRIPTION
**Pull request by icewind1991**
cc @PVince81 @DeepDiver1975

### CODE REVIEW

File: lib/private/files/cache/cache.php [hunk 1]
```diff
@@ -137,8 +162,28 @@ public function loadMimetypes() {
 	/**
 	 * get the stored metadata of a file or folder
 	 *
-	 * @param string /int $file
-	 * @return array|false
+	 * the returned cache entry contains at least the following values:
+	 * [
+	 * 		'fileid' => int, the numeric id of a file (see getId)
```
Comment (DeepDiver1975) [inline lib/private/files/cache/cache.php]: 'encrypted' is missing

Comment (icewind1991) [inline lib/private/files/cache/cache.php]: > `at least`

I'm not sure about all the uses of the encrypted field and if all cache implementations are required to provide it so I didn't feel comfortable about adding it there

**Comment by DeepDiver1975** [comment]
:+1:

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**